browser-test/src/support/ti_dashboard.ts

```diff
@@ -2,7 +2,8 @@ import {Page} from 'playwright'
 
 /*
  * This class is to test Civiform in the TI path
- * It impements functionality to create a new client,Test if the given client is present in the dashboard
+ * It impements functionality to create a new client,Test if the given client is present/not present in the dashboard
+ * It can also update a Client's Data of Birth to a new one
```

Comment:
type?

Reply:
@syntapy - Can you please approve the PR if you don't have further comments?